controller/PciManager.php

```diff
@@ -253,6 +271,28 @@ protected function getRequestPciDataObject()
         return null;
     }
 
+
+    /**
+     * @throws PortableElementException
+     * @throws PortableElementVersionIncompatibilityException
+     * @throws \common_Exception
+     * @throws \common_exception_Error
+     * @throws \HttpRequestException
+     */
+    public function unregister()
+    {
+        $pci = $this->getRequestPciDataObject();
```

Comment:
shouldn't we surround with `try/catch` to capture errors and server `success => false` in case of errors ?